LDAR_Sim/src/initialization/initialize_emissions.py

```diff
@@ -78,7 +80,7 @@ def initialize_emissions(
     return None
 
 
-def read_in_emissions(infrastructure, generator_dir, sim_numb):
+def read_in_emissions(infrastructure: Infrastructure, generator_dir, sim_numb):
```

Comment:
If typing is added for the infrastructure, generator_dir, and sim_numb should also have their types stated to keep it consistent.

Reply:
Yup, good catch